interpreter/subnet/src/main/scala/io/buoyant/transformer/SubnetGatewayTransformer.scala

```diff
@@ -63,7 +63,10 @@ class GatewayTransformer(
           // select the gateway addresses that share a subnet with addr
           gatewayAddrs.filter(gatewayPredicate(addr, _))
         }
-        Addr.Bound(selected, meta)
+        if (selected.isEmpty)
+          Addr.Neg
```

Comment:
Right, it could be extremely useful or extremely annoying. I'm fine leaving as is